tasks/discover-granules/index.js

```diff
@@ -285,12 +285,9 @@ const discoverGranules = async ({ config }) => {
  *
  * @param {Object} event - a Cumulus Message
  * @param {Object} context - an AWS Lambda context
- * @param {Function} callback - an AWS Lambda handler
  * @returns {undefined} - does not return a value
```

Comment:
returns a Promise now

Reply:
This is going to be true for all of these handlers

Reply:
Argh.  Yes, you're correct, I thought I'd fixed that.     Good catch.

Reply:
478d33f08

Most of them could reference the schema/output.json.  That seemed sufficient.

Reply:
(and 44b449f08)